tsconfig.spec.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./out-tsc/spec",
+    "types": ["jest", "node"]
+  },
+  "files": ["src/polyfills.ts"],
+  "include": ["src/tests/*.spec.ts", "src/**/*.d.ts"]
```

Comment:
Os testes estarão no próprio diretório do módulo/componente/página, não?
Se for o caso, esse caminho está incorreto.